azurelinuxagent/agent.py

```diff
@@ -66,9 +66,6 @@ def __init__(self, verbose, conf_file_path=None):
         level = logger.LogLevel.VERBOSE if verbose else logger.LogLevel.INFO
         logger.add_logger_appender(logger.AppenderType.FILE, level,
                                    path=conf.get_agent_log_file())
-        if conf.get_logs_console():
```

Comment:
Did we get some communication out to EEE people? I think they are dependent on this